universe/info.owlbot/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "info.owlbot",
+  "version": "1.0.0",
+  "description": "A basic dictionary interface to generate info about words.",
+  "author": "Aydan Pirani",
+  "main": "device.js",
+  "dependencies": ["owlbot-js"]
```

Comment:
Is this the right format? I'm not sure it is.
Typically, one uses `npm install --save` to change the dependencies in the package.json, rather than editing by hand.